core/core.go

```diff
@@ -217,10 +225,16 @@ func New(config Config) (*App, error) {
 	}
 
 	// Initialize ETH JSON-RPC RateLimiter
-	clock := clock.New()
-	ethRPCRateLimiter, err := ratelimit.New(config.EthereumRPCMaxRequestsPer24HrUTC, config.EthereumRPCMaxRequestsPerSecond, meshDB, clock)
-	if err != nil {
-		return nil, err
+	var ethRPCRateLimiter ratelimit.RateLimiter
+	if config.EnableEthereumRPCRateLimiting == false {
+		ethRPCRateLimiter = ratelimit.NewUnlimited()
```

Comment:
@fabioberger I opted to re-use the existing `fakeLimiter` type and just renamed the constructor from `NewFakeLimiter` to `NewUnlimited`. This way we can still count and log the number of RPC requests that were allowed. It also required the fewest code changes. Let me know if there would be any issues with this approach.